backend/app/Models/Transaction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Transaction extends Model
+{
+    const FIELD_TRANSACTION_TIME = 'transaction_time';
+
+    public function transactionCard()
+    {
+        return $this->hasOne(Card::class, 'id', 'card_id');
```

Comment:
`Card::class` is not in the scope of this file. Use `'App\Models\Card'` for the relationship.

Reply:
no `Card::class` is OK and works because `Transaction` and `Card` are both in the `Models` namespace